source/adios2/core/ADIOS.cpp

```diff
@@ -79,6 +79,18 @@ IO &ADIOS::DeclareIO(const std::string ioName)
     return ioPair.first->second;
 }
 
+IO &ADIOS::GetIO(const std::string name)
+{
+    auto itIO = m_IOs.find(name);
+    if (itIO == m_IOs.end())
```

Comment:
m_DebugMode = true --> we try to catch as many exceptions as possible from user input
m_DebugMode = false --> the user has run the same thing over and over again and it works, no need for the same checks over and over again.